addons/sale_project/models/sale_order_line.py

```diff
@@ -24,7 +24,7 @@ class SaleOrderLine(models.Model):
 
     def name_get(self):
         res = super().name_get()
-        with_price_unit = self.env.context.get('with_price_unit')
+        with_price_unit = self._context.get('with_price_unit', self._context.get('is_timesheet'))
```

Comment:
That was the idea I got. Or is it a bad idea?
```suggestion
        with_price_unit = self._context.get('is_timesheet')
```

What do you think @xavierbol ?

Reply:
Hello @kcv-odoo after having thought of it I think that your proposition was right. I actually thought it would be possible to get rid of the with_price_unit context key but it finally seems after having talked to @xavierbol  that this will not be possible.
Can you revert to ?
```python
with_price_unit = self._context.get('with_price_unit', self._context.get('is_timesheet'))
```

Thanks and sorry for the inconvenience 🙈